### ISSUE TITLE
Outline view does not seem to recognize all moonscript functions

### ISSUE THREAD

**Issue by yumaikas** [nesbox/TIC-80#2237]
When I write moonscript functions that aren't `fn_name=->`, they aren't picked up by the outline on the 1.0.2164 build.

Based on what I can see, it's specifically `=->` that tags a function to be included in the outline view.

**Comment by Skeptim** [comment]
Indeed, that's here: https://github.com/nesbox/TIC-80/blob/0602a612654fc1e57741917cf2353da336bc6c5d/src/api/moonscript.c#L157

**Comment by joshgoebel** [comment]
Can you provide examples of the various ways one defines functions in moonscript?
